lib/organisms/flexible-content/flexible-content.directive.js

```diff
@@ -39,7 +39,7 @@ function lnOFlexibleContent($compile, $injector, $sanitize) {
             parAttr = prefix + parAttr;
           }
 
-          var value = angular.isObject(row[key]) ? $sanitize(angular.toJson(row[key])) : row[key];
+          var value = angular.isObject(row[key]) ? $sanitize(angular.toJson(row[key])) : row[key].replace(/"/g, '\'');
```

Comment:
No @mitogh , just with double quotes because they break the attribute content :(